spec/src/main/asciidoc/servlet-spec-body.adoc

```diff
@@ -8546,6 +8546,9 @@ Jakarta Servlet {spec-version} specification developed under the Jakarta EE Work
 
 === Changes Since Jakarta Servlet 5.0
 
+link:https://github.com/eclipse-ee4j/servlet-api/issues/433[Issue 433]::
+Clarify how IO errors are handled by async streams.
+
```

Comment:
This needs moving to the correct section

Reply:
Moved